ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
> I don't know why but I suppose that tinyMCE use the p tag as "marker" to know stuffs (where to insert new text, where to display or not the insert toolbar, ...).

It wouldn't wrap a  `<div>` in a `<p>`.

What they mean by "it cripples the editor" is that the editor needs block elements at the root, not inline ones.

So with the migration you're not doing the same as what TinyMCE would do, since you're unconditionally wrapping everything in a `<p>`, even block elements.

In addition, we currently don't have any HTML in these fields, so we won't have any inline elements at the root, just text. And we won't have any elements inside that text either, so the editor should not have any problem.

But you know what? I've just tried it. :)

I had a blog post with `This is a test summary` as its summary. And I checked out your branch, without running the migration, and ran the server.

When loading the form to edit the blog post, TinyMCE automatically wrapped the summary in `<p>`. It did it when **displaying** the form, before I ever saved in with TinyMCE.

This is what I said in my first comment here: that I've seen other rich-text editors do this, and I strongly suspected TinyMCE would do the same.

So I maintain that you shouldn't wrap in `<p>` in the migration, only `escape()`.